### PR TITLE
 Added deployment section for the systemlock package - (Solves - #42)

### LINKED ISSUE #42: Deployment of sddm package as a github release
- Issue type: Feature request 

### Short description

For usage of systemlock package in the meilix main repo, we need to deploy the systemlock package as a github release(one of the possible solution, another will be deploying the systemlock package on the new deb branch)

### PR DESCRIPTION
**Pull request by pranav1698**
### Short description
This adds deployment of the systemlock package as a github release
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`

### For the reviewers
I have:
- [ ] Reviewed this pull request as an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

Fixes #42

### CODE REVIEW

**Comment by pranav1698** [comment]
@abishekvashok, @meets2tarun @tabesin please review 
Another approach would be opening up the deb branch fro the deployment there

File: .travis.yml [hunk 1]
```diff
@@ -14,3 +14,15 @@ script:
...
 notifications: 
 slack: fossasia:3pMkQrSogvEMfnu9aJDW2ZYC
+before_deploy:
+- export lock_package="`echo systemlock_*_all.deb`"
+- cat systemlock_*_all.deb > "/home/travis/${lock_package}"
+- cp log.txt /home/travis/log.txt
+deploy:
+  provider: releases
+  api_key:
+    secure:
```
Comment (abishekvashok) [inline .travis.yml]: We need a secure key

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (pranav1698) [reply .travis.yml]: We need a key from one of the project maintainers

Comment (abishekvashok) [reply .travis.yml]: If you could Travis dashboard it, instead of a secure key, I could provide one

Comment (pranav1698) [reply .travis.yml]: @abishekvashok I have removed the security_key now we can add the github oauth token for the meilix-systemlock project

**Comment by pranav1698** [comment]
@tabesin @abishekvashok  please review

File: .travis.yml [hunk 2]
```diff
@@ -14,3 +14,14 @@ script:
 - "./debuild.sh |& tee log.txt"
 notifications: 
 slack: fossasia:3pMkQrSogvEMfnu9aJDW2ZYC
+before_deploy:
+- export lock_package="`echo systemlock_*_all.deb`"
+- cat systemlock_*_all.deb > "/home/travis/${lock_package}"
+- cp log.txt /home/travis/log.txt
```
Comment (abishekvashok) [inline .travis.yml:20]: We aren't deploying it any reason to copy it?

File: .travis.yml [hunk 3]
```diff
@@ -14,3 +14,14 @@ script:
...
+deploy:
+  provider: releases
+  api_key:
```
Comment (abishekvashok) [inline .travis.yml:23]: We need an api key

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by pranav1698** [comment]
Let me open a pr for the master branch

**Pull request closed without merging**